src/App.tsx

```diff
@@ -35,7 +41,15 @@ const AppComponent: React.FC<AppProps> = () => {
         <ButtonComponent variant={ButtomVariants.s}>click me</ButtonComponent>
         <ButtonComponent variant={ButtomVariants.xs}>click me</ButtonComponent>
         <button onClick={() => dispatch(fetchTheme())}>Refresh theme</button>
+        <button onClick={() => dispatch(loginUser())}>Login User</button>
       </div>
+      <Routes>
+        <Route path="/dashboard" element={<Dashboard userName={isFetchingUser ? 'Logging In' : user}/>} >
```

Comment:
Why is user passed as props to dashboard? I think dashboard can be connected to redux and pick user state directly from redux.